packages/canary-web-components/src/components/ic-card-horizontal/ic-card-horizontal.tsx

```diff
@@ -0,0 +1,249 @@
+import {
+  Component,
+  Element,
+  Listen,
+  Prop,
+  State,
+  h,
+  Method,
+} from "@stencil/core";
+import {
+  onComponentRequiredPropUndefined,
+  isSlotUsed,
+  getThemeFromContext,
+  removeDisabledFalse,
+} from "../../utils/helpers";
+import {
+  IcTheme,
+  IcThemeForeground,
+  IcThemeForegroundEnum,
+} from "../../utils/types";
+import { IcCardSizes } from "./ic-card-horizontal.types";
+
+/**
+ * @slot heading - Content will be placed at the top of the horizontal card to the right of the icon.
+ * @slot message - Content will be placed in the main body of the horizontal card.
+ * @slot image - Content will be placed to the left of all other content.
+ * @slot icon - Content will be placed to the left of the horizontal card heading.
+ * @slot badge - Badge component overlaying the top right of the horizontal card.
+ */
+@Component({
+  tag: "ic-card-horizontal",
```

Comment:
Why did you opt to call it "ic-card-horizontal" as opposed to "ic-horizontal-card"?

Reply:
A member of the community suggested it, alongside a name change to ic-card to make it ic-card-vertical, so that they still appear next to each other alphabetically. I created a ticket for that work - https://github.com/mi6/ic-ui-kit/issues/2216